src/utils/convertToMoment.js

```diff
@@ -19,6 +19,10 @@ export default (newProps, momentProps) => {
                         undefined
                 );
             }
+        } else if (Array.isArray(value)) {
+            dest[key] = value.map(d =>
+                d === null || d === undefined ? null : moment(d)
```

Comment:
Might not even need the null case here since we're in an array with propType string (the single values for max and min need that test because they're optional props)

But if we do keep it, you can use https://ramdajs.com/docs/#isNil

Reply:
Thanks, I opted for ditching the check since it's nice and clean: `value.map(moment)`

Reply:
Turns out that actually causes some problems, but `value.map(d => moment(d))` is ok.